F1/F1/Config/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/02.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
```

Comment:
unnecessary new lines

Reply:
also is this file being used? It doesn't look like it? So you can remove it please